src/csharp/build_packages.bat

```diff
@@ -8,7 +8,7 @@ set PROTOBUF_VERSION=3.0.0-beta2
 set VERSION_WITH_BETA=%VERSION%-beta
 
 @rem Adjust the location of nuget.exe
-set NUGET=C:\nuget\nuget.exe
```

Comment:
Will do. I assumed that nuget would be available in $(SolutionDir)/.nuget/nuget.exe, I will revert this change.